ddtrace/internal/_encoding.pyx

```diff
@@ -524,7 +525,7 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
     cdef void * get_dd_origin_ref(self, str dd_origin):
         return string_to_buff(dd_origin)
 
-    cdef inline int _pack_meta(self, object meta, char *dd_origin) except? -1:
+    cdef inline int _pack_meta(self, object meta, char *dd_origin, int i) except? -1:
```

Comment:
same here, instead of `i` maybe a more explicit name like `bool include_global_meta`